platform/broadcom/sonic-platform-modules-ruijie/debian/platform-modules-ruijie-b6510-48vs8cq.install

```diff
@@ -0,0 +1,3 @@
+b6510-48vs8cq/modules/sonic_platform-1.0-py2-none-any.whl /usr/share/sonic/device/x86_64-ruijie_b6510-48vs8cq-r0
+b6510-48vs8cq/scripts/b6510-48vs8cq_platform.sh /usr/local/bin
+b6510-48vs8cq/systemd/platform-modules-b6510-48vs8cq.service /etc/systemd/system
```

Comment:
@jleveque, I remember we are adding all service to /lib/systemd, not /etc/systemd/system directly?

Reply:
We recently migrated all service files from /etc/systemd/system/ to **/usr/lib/systemd/system/**

Reply:
ok. will change the service path